.github/workflows/build.yml

```
@@ -281,6 +281,51 @@ jobs:
             ../scripts/xcodebuild.sh macos/TemplateExample.xcworkspace build
           fi
         working-directory: template-example
+  windows:
+    name: "Windows"
+    runs-on: windows-latest
+    strategy: 
+      matrix:
+        configuration: [Debug, Release]
+        platform: [ARM, x64, x86]
```

I think we can skip x86...
```suggestion
        platform: [ARM, x64]
```